reducers/views/threads.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {combineReducers} from 'redux';
+
+import {GenericAction} from 'mattermost-redux/types/actions';
+
+import {ViewsState} from 'types/store/views';
+
+import {Threads} from 'utils/constants';
+
+export const selectedThreadIdInTeam = (state: ViewsState['threads']['selectedThreadIdInTeam'] | null = null, action: GenericAction) => {
+    if (action.type === Threads.CHANGED_SELECTED_THREAD) {
+        return {
+            ...state,
+            [action.data.team_id]: action.data.id,
```

Comment:
nit: should we be using `team_id` and `post_id` or some such to clearly explain what the `id` is?

Reply:
Even though I think it's ok when the subject is small and clear, I do think in many cases it's good to avoid generic names.